doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1576,6 +1717,22 @@ Controller subsystem for nRF5340 Audio
 
 The following known issues apply to the LE Audio subsystem (NET core controller) for nRF5340 used in the nRF5340 Audio application.
 
+.. rst-class:: v2-5-0
```

Comment:
New issue for Controller subsystem for nRF5340 Audio:

- - -
Affected versions: NCS 2.5.0
OCT-2761: Controller asserts on CIS gateway if two paused headsets reset

If a CIS headset pair is paused, then reset simultaneously a few times while remaining paused, the controller on gateway might get unresponsive.

Reply:
Updated. Please check.

Reply:
I wrote the wrong ticket number here. It should be OCX-223. Apologize

Reply:
Updated. Please check.